.travis.yml

```diff
@@ -12,38 +12,26 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env:
-        - DEPS=lowest
-    - php: 5.6
-      env:
-        - DEPS=latest
-    - php: 7
-      env:
-        - DEPS=lowest
-    - php: 7
-      env:
-        - DEPS=latest
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPS=lowest
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPS=latest
-        - CS_CHECK=true
-        - TEST_COVERAGE=true
-    - php: 7.2
+    - php: 7.4
       env:
         - DEPS=lowest
-    - php: 7.2
+    - php: 7.4
       env:
         - DEPS=latest
-    - php: 7.3
+    - php: 'nightly'
       env:
         - DEPS=lowest
-    - php: 7.3
+        - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
```

Comment:
If `laminas/laminas-coding-standard` goes to `2.2` then probably we don't need this ignore

Reply:
I was able to install dependencies on PHP 8 just fine even though [1.0.0](https://github.com/laminas/laminas-json/blob/185ec9ffaf10427da0f3ad3fd42d8646ff934a9c/composer.json#L32) is required. I'll remove this now.